stable/external-dns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.14.3
+version: 2.14.4
```

Comment:
As it is a new feature and not bug fixing I would increase this in a minor
```suggestion
version: 2.15.0
```

Reply:
Done.